src/ape/api/query.py

```diff
@@ -0,0 +1,108 @@
+import sys
```

Comment:
I think this whole file might currently be excluded from the docs and you may need to add some stuff there